client/src/modules/reports/generate/stock_value/stock_value.config.js

```diff
@@ -100,6 +88,9 @@ function StockValueConfigController($sce, Notify, SavedReports,
   };
 
   function checkCachedConfiguration() {
-    vm.reportDetails = angular.copy(cache.reportDetails || {});
+    if (cache.reportDetails) {
+      vm.reportDetails = angular.copy(cache.reportDetails);
+      vm.dateTo = new Date(); // always default to today
+    }
```

Comment:
Need to add a default for the currency here.   See the checkCachedConfiguration() function in client/src/modules/reports/generate/avg_med_costs_per_patient/avg_med_costs_per_patient.js for an example.

Reply:
Done.